Gems/Atom/RPI/Code/Include/Atom/RPI.Public/GpuQuery/GpuPassProfiler.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/std/containers/array.h>
+#include <AzCore/Name/Name.h>
+
+#include <Atom/RPI.Public/GpuQuery/GpuQueryTypes.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
+        class Pass;
+        class ParentPass;
+
+        //! This is a helper class that can be used to measure, per frame, the time it takes
+        //! for the render pipeline to execute in the GPU from the first to the last pass.
```

Comment:
The "render pipeline" term could be confusing since the RPI has RenderPipeline which may only represent part of the frame graph. 
I would suggest using "rendering frame" instead.

Reply:
Sounds good.